cluster/manager/manager.go

```diff
@@ -1084,7 +1102,7 @@ func (c *ClusterManager) waitForQuorum(exist bool) error {
 			break
 		} else {
 			c.status = api.Status_STATUS_NOT_IN_QUORUM
-			if quorumRetries == 600 {
+			if quorumRetries == maxQuorumRetries {
 				err := fmt.Errorf("Unable to achieve Quorum." +
 					" Timeout 20 minutes exceeded.")
```

Comment:
This log message might be misleading if the timeout has been changed from the default value of 20 min.

Reply:
fixed